src/main/webapp/isolate/registration.jsp

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Registration</title>
+</head>
+<body>
+  <div class="container" id="container">
+    <p>
+      Please fill out this form if you are someone who is isolating as a result
+      of COVID-19, and would like to be matched to a volunteer.
```

Comment:
I would rather state that they'd need help from a volunteer, as they'd probably be matched more than once

Reply:
https://github.com/googleinterns/step226-2020/pull/4/commits/6f8cfebff35c97ca3a285cced0d5b5512a45ce2f